source/georef/georef.cpp

```diff
@@ -301,11 +301,11 @@ ProjectionData::ProjectionData(const type::GeographicalCoord & coord, const GeoR
     }
 }
 
-ProjectionData::ProjectionData(const type::GeographicalCoord & coord, const GeoRef & sn, type::idx_t offset, const proximitylist::ProximityList<vertex_t> &prox){
+ProjectionData::ProjectionData(const type::GeographicalCoord & coord, const GeoRef & sn, type::idx_t offset, const proximitylist::ProximityList<vertex_t> &prox, double distance){
     edge_t edge;
     found = true;
     try {
-        edge = sn.nearest_edge(coord, prox, offset);
+        edge = sn.nearest_edge(coord, prox, offset, distance);
```

Comment:
Ha, you talk about the name of the parameter `distance`... `search_radius` or `horizon` is OK to me.